ella_flatcomments/forms.py

```diff
@@ -15,6 +15,7 @@ class FlatCommentForm(ModelForm):
     def __init__(self, content_object, user, *args, **kwargs):
         self.content_object = content_object
         self.user = user
+        self.request = kwargs.pop('request', None)
```

Comment:
it might be better to have the `request` as optional (or even required) arg to `post` method on the form, will ceraintly make the code a lot cleaner.